scrapinghub/client/collections.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import absolute_import
 import collections
+import collections.abc
 
 from six import string_types
```

Comment:
Should we remove this as part of this pull request, or will we handle the post-Python-2 cleanup separately?